allure-cucumber4-jvm/src/main/java/io/qameta/allure/cucumber4jvm/LabelBuilder.java

```diff
@@ -0,0 +1,135 @@
+
+                // Handle composite named links
+                if (tagKey.startsWith(PLAIN_LINK + ".")) {
+                    tryHandleNamedLink(tagString);
+                    continue;
+                }
+
+                switch (tagKey) {
+                    case SEVERITY:
+                        getScenarioLabels().add(createSeverityLabel(tagValue.toLowerCase()));
+                        break;
+                    case TMS_LINK:
+                        getScenarioLinks().add(createTmsLink(tagValue));
+                        break;
+                    case ISSUE_LINK:
+                        getScenarioLinks().add(createIssueLink(tagValue));
+                        break;
+                    case PLAIN_LINK:
+                        getScenarioLinks().add(createLink(null, tagValue, tagValue, null));
+                        break;
+                    default:
+                        LOGGER.warn("Composite tag {} is not supported. adding it as RAW", tagKey);
+                        getScenarioLabels().add(getTagLabel(tag));
+                        break;
+                }
+            } else if (tagParser.isPureSeverityTag(tag)) {
+                getScenarioLabels().add(createSeverityLabel(tagString.substring(1)));
+            } else if (!tagParser.isResultTag(tag)) {
+                getScenarioLabels().add(getTagLabel(tag));
+            }
+        }
+
+        getScenarioLabels().add(new Label().setName("host").setValue(getHostName()));
+        getScenarioLabels().add(new Label().setName("package").setValue(feature.getName()));
+        getScenarioLabels().add(new Label().setName("suite").setValue(feature.getName()));
+        getScenarioLabels().add(new Label().setName("testClass").setValue(scenario.getName()));
+        getScenarioLabels().add(new Label().setName("thread").setValue(getThreadName()));
+
+    }
+
+    public List<Label> getScenarioLabels() {
+        return scenarioLabels;
+    }
+
+    public List<Link> getScenarioLinks() {
+        return scenarioLinks;
+    }
+
+    private Label getTagLabel(final PickleTag tag) {
+        return createTagLabel(tag.getName().substring(1));
+    }
+
+    /**
+     * Handle composite named links.
+     *
+     * @param tagString Full tag name and value
+     */
+    private void tryHandleNamedLink(final String tagString) {
+        final String namedLinkPatternString = PLAIN_LINK + "\\.(\\w+-?)+=(\\w+(-|_)?)+";
+        final Pattern namedLinkPattern = Pattern.compile(namedLinkPatternString, Pattern.CASE_INSENSITIVE);
+
+        if (namedLinkPattern.matcher(tagString).matches()) {
+            final String type = tagString.split(COMPOSITE_TAG_DELIMITER)[0].split("[.]")[1];
+            final String name = tagString.split(COMPOSITE_TAG_DELIMITER)[1];
+            getScenarioLinks().add(createLink(null, name, null, type));
+        } else {
+            LOGGER.warn("Composite named tag {} is not matches regex {}. skipping", tagString,
```

Comment:
Should say "Composite named tag {} does not match regex {}. skipping"

Reply:
thanks, will change